lib/jellyfish/room.ex

```diff
@@ -356,4 +365,8 @@ defmodule Jellyfish.Room do
   end
 
   defp registry_id(room_id), do: {:via, Registry, {Jellyfish.RoomRegistry, room_id}}
+
+  defp check_video_codec(:h264, Jellyfish.Component.HLS), do: :ok
+  defp check_video_codec(_codec, Jellyfish.Component.HLS), do: {:error, :wrong_codec}
```

Comment:
[nitpick] imo this sounds better in the technical context
```suggestion
  defp check_video_codec(_codec, Jellyfish.Component.HLS), do: {:error, :invalid_codec}
```